requirements.txt

```diff
@@ -6,4 +6,4 @@ ipaddress>=1.0.22
 absl-py
 ply
 mock
-six # requirement from absl-py
+six>=1.12.0
```

Comment:
I think it'd be good to use frozen package versions in the requirements file to avoid future similar issues and help with duplicating environments, but that's outside of the scope of this PR